link_shortener/view_routes.py

```diff
@@ -68,7 +69,14 @@ async def about_page(request):
 @login_required
 @credential_whitelist_check
 async def all_active_links(request, user):
-    link_data = await retrieve_links(request, filters={'is_active': True})
+    filter_set = {'is_active', 'owner', 'search'}
```

Comment:
Proč je to set? Těch filtrů doufám nikdy nebude tolik abychom to museli hlídat proti duplicitám. Určitě to není špatně, jen mě to zajímá

Reply:
Bez dovodu 🤷‍♂️ :) Ze vraj sety su rychlejsie, asi preto.